src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -135,12 +155,140 @@ def load_arguments(self, _):
                    completer=get_resource_name_completion_list('Microsoft.Web/serverFarms'),
                    configured_default='appserviceplan', id_part='name', local_context_attribute=None)
 
+    with self.argument_context('appservice plan managed-instance install-script') as c:
+        c.argument('name', arg_type=name_arg_type, help='The name of the app service plan',
+                   completer=get_resource_name_completion_list('Microsoft.Web/serverFarms'),
+                   configured_default='appserviceplan', id_part='name',
+                   local_context_attribute=LocalContextAttribute(name='plan_name', actions=[LocalContextAction.GET]))
+        c.argument('resource_group_name', arg_type=resource_group_name_type)
+
+    with self.argument_context('appservice plan managed-instance install-script add') as c:
+        c.argument('install_script_name', options_list=['--install-script-name'], help='Name of the install script')
+        c.argument('source_uri', options_list=['--source-uri'], help='Source URI of the install script')
+        c.argument('install_script_type', options_list=['--type'], help='Type of the install script', arg_type=get_enum_type(INSTALL_SCRIPT_TYPES))
+
+    with self.argument_context('appservice plan managed-instance install-script remove') as c:
+        c.argument('install_script_name', options_list=['--install-script-name'], help='Name of the install script to remove')
+
+    with self.argument_context('appservice plan managed-instance install-script list') as c:
+        c.ignore('ids')
+        c.argument('name', id_part=None)
+
+    with self.argument_context('appservice plan managed-instance storage-mount') as c:
+        c.argument('name', arg_type=name_arg_type, help='The name of the app service plan',
+                   completer=get_resource_name_completion_list('Microsoft.Web/serverFarms'),
+                   configured_default='appserviceplan', id_part='name',
+                   local_context_attribute=LocalContextAttribute(name='plan_name', actions=[LocalContextAction.GET]))
+        c.argument('resource_group_name', arg_type=resource_group_name_type)
+
+    with self.argument_context('appservice plan managed-instance storage-mount list') as c:
+        c.ignore('ids')
+        c.argument('name', id_part=None)
+
+    with self.argument_context('appservice plan managed-instance storage-mount add') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type)
+        c.argument('name', arg_type=name_arg_type, help='The name of the app service plan')
+        c.argument('mount_name', options_list=['--mount-name'], help='Name of the storage mount')
+        c.argument('mount_type', options_list=['--type'], arg_type=get_enum_type(STORAGE_MOUNT_TYPES), help='Type of the storage mount')
+        c.argument('source', options_list=['--source'], help='Source URI or path for the storage mount')
```

Comment:
just a reminder: you don’t need to specify options_list if the parameter name is exactly the same as the argument name.